modules/exploits/windows/local/unquoted_service_path.rb

```diff
@@ -30,17 +32,22 @@ def initialize(info={})
 
         The offensive technique is also described in Writing Secure Code (2nd Edition),
         Chapter 23, in the section "Calling Processes Security" on page 676.
+
+        This technique was previously called Trusted Service Path, but is more commonly
+        known as Unquoted Service Path.
```

Comment:
Could you please add a note that the executable that is written to disk will require manual cleanup? The exploited service won't work until it's cleaned up as well which could be problematic if left in that state.

Reply:
added a note to info, module docs, and a print about it.